src/model_signing/signing.py

```diff
@@ -16,7 +16,7 @@
 
 Users should use this API to sign models, rather than using the internals of the
 library. We guarantee backwards compatibility only for the API defined in
-`hash.py`, `sign.py` and `verify.py` at the root level of the library.
+`hashing.py`, `sign.py` and `verify.py` at the root level of the library.
```

Comment:
all three files changed names right?

Reply:
Right, I forgot that. Should be fixed now